src/server/logging-middleware.ts

```diff
@@ -1,22 +1,20 @@
-import pinoHttp from "pino-http";
+import pinoHttp, { startTime, Options, HttpLogger } from "pino-http";
```

Comment:
Better to use type imports
```suggestion
import pinoHttp, { startTime, type Options, type HttpLogger } from "pino-http";
```